authors/apps/articles/views.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from .models import Article
+from django.shortcuts import get_object_or_404
+from .serializers import ArticleSerializer, ArticleUpdateSerializer
+
+class ArticleListCreateView(generics.ListCreateAPIView):
+    queryset = Article.objects.all()
+    serializer_class = ArticleSerializer
+    permission_classes = (IsAuthenticatedOrReadOnly,)
```

Comment:
does this support getting an a single article

Reply:
@MuhweziDeo there is actually another generic view method that does the retrieve update and delete just below this one